tests/by-util/test_dd.rs

```diff
@@ -1536,3 +1536,32 @@ fn test_multiple_processes_reading_stdin() {
         .succeeds()
         .stdout_only("def\n");
 }
+
+#[cfg(unix)]
+#[test]
+fn test_seekable_stdin() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+    at.write("f", "abcd");
+    ts.ucmd()
+        .set_stdin(File::open(at.plus("f")).unwrap())
+        .args(&["bs=1", "skip=5", "count=0", "status=noxfer"])
+        .run()
+        .stderr_is("dd: 'standard input': cannot skip: Invalid argument\n0+0 records in\n0+0 records out\n")
+        .code_is(1);
+}
+
+#[cfg(unix)]
+#[test]
+fn test_seekable_stdout() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+    at.write("f", "abcd");
+    ts.ucmd()
+        .pipe_in("abcd")
+        .set_stdout(File::open(at.plus("f")).unwrap())
+        .args(&["bs=1", "seek=10000000000000000000", "count=0", "status=noxfer"])
+        .run()
+        .stderr_is("dd: 'standard output': cannot seek: Invalid argument\n0+0 records in\n0+0 records out\n")
```

Comment:
For this case I get from GNU:
```
dd: invalid number: ‘10000000000000000000’: Value too large for defined data type
```
Which is more informative and has nothing to do with the length of the target file.

Reply:
Hey, sorry was a little bit busy last week. The mentioned test is a "root test" and can be found here: `tests/dd/skip-seek-past-dev.sh`. It takes a block device as stdin/-out and therefore requires root. I try to "simulate" this by using a normal file, which unfortunately behaves not quite similar.

Reply:
So the error message should come when the position to skip to is "past" the device. My idea was to take a "high enough number" to skip past "every" device, but it seem to work not always as expected.

Reply:
I guess I will adapt my implementation to be more specific and only work with block devices (which I can find out via stat).